playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -172,56 +172,56 @@
     - ingress
     - run
 
-- name: Template file for the namespaces ingress
-  config_template:
-    src: ingress-namespace.yaml.j2
-    dest: /tmp/socok8s-ingress-namespace.yaml
-    config_overrides: "{{ suse_osh_deploy_ingress_namespace_yaml_overrides }}"
-    config_type: yaml
-
 - name: Setup Basic ingress and namespace
   import_tasks: component-install.yml
   vars:
-    _cmpnt_template: ingress-kube-system.yaml
-    _cmpnt_overrides: "{{ suse_osh_deploy_ingress_kube_system_yaml_overrides }}"
     _cmpnt_runcommand: ./tools/deployment/multinode/020-ingress.sh
-  environment:
-    OSH_EXTRA_HELM_ARGS_INGRESS_KUBE_SYSTEM: "{{ lookup('env', 'OSH_EXTRA_HELM_ARGS_INGRESS_KUBE_SYSTEM') | default('', True) }} --values /tmp/socok8s-ingress-kube-system.yaml"
-    OSH_EXTRA_HELM_ARGS_INGRESS_OPENSTACK: "{{ lookup('env', 'OSH_EXTRA_HELM_ARGS_INGRESS_OPENSTACK') | default('', True) }} --values /tmp/socok8s-ingress-namespace.yaml"
-    OSH_EXTRA_HELM_ARGS_INGRESS_CEPH: "{{ lookup('env', 'OSH_EXTRA_HELM_ARGS_INGRESS_CEPH') | default('', True) }} --values /tmp/socok8s-ingress-namespace.yaml"
+    _cmpnt_overrides:
+      - suse: ingress-namespace.yaml
+        user: "{{ suse_osh_deploy_ingress_namespace_yaml_overrides }}"
+        envvars_getting_those_overrides:
```

Comment:
Why such cryptic variable name?

Reply:
Could you propose an alternative?

Reply:
`extra_overrides`?

Reply:
it's not extra overrides, it's to know where the overrides apply.

Reply:
Used different name, should be clearer.